airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/SQLNamingResolvable.java

```diff
@@ -24,10 +24,14 @@
 
 package io.airbyte.integrations.base;
 
-public class NamingHelper {
+import com.fasterxml.jackson.databind.JsonNode;
 
-  public static String getRawTableName(String streamName) {
-    return streamName + "_raw";
-  }
+public interface SQLNamingResolvable {
+
+  String getRawSchemaName(JsonNode config, String schemaName, String streamName);
+
+  String getRawTableName(JsonNode config, String streamName);
```

Comment:
for all of these method can you add a javadocs style comment that describes the contract of these methods. e.g. 
```
  /**
   * This method converts the streamName, which can be any UTF8 encoded string, into a string 
   * that the destination can use as a table name.
   * @param config
   * @param streamName
   * @return cleaned table name
   */
```

If you start typing `/**` into intellij above a method name it'll autocomplete most of it for you :D